SemiconductorTestLibrary.Extensions/source/InstrumentAbstraction/DCPower/Source.cs

```diff
@@ -160,11 +220,27 @@ public static void ForceVoltage(this DCPowerSessionsBundle sessionsBundle, IDict
             {
                 var perPinSettings = settings[sitePinInfo.PinName];
                 perPinSettings.OutputFunction = DCPowerSourceOutputFunction.DCVoltage;
-                perPinSettings.LimitSymmetry = DCPowerComplianceLimitSymmetry.Symmetric;
                 sessionInfo.Force(perPinSettings, sitePinInfo.IndividualChannelString, waitForSourceCompletion);
             });
         }
 
+        /// <summary>
+        /// Forces voltage and specifies current limit.
```

Comment:
Suggest changing to: "Forces voltage using specified pin- and site-unique source settings."